examples/natural_language_classifier_v1.py

```diff
@@ -18,8 +18,12 @@
 
 # create a classifier
 # with open('../resources/weather_data_train.csv', 'rb') as training_data:
-#     print(json.dumps(natural_language_classifier.create(
-# training_data=training_data, name='weather'), indent=2))
+#     metadata = json.dumps({'name': 'my-classifier', 'language': 'en'})
+#     classifier = natural_language_classifier.create_classifier(
+#         metadata=metadata,
```

Comment:
this should be training_metadata now, right?

Reply:
`training_metadata` is what we name the parameter when we send it to the service, but the parameter is still called `metadata` for your purposes.

Reply:
gotcha, cool cool